modules/ROOT/pages/business-groups.adoc

```diff
@@ -1,171 +1,74 @@
 
-If you do not have the Organization Administrator permission, you can view only the business groups that you have permissions to view. In the *Organization* tab, your organization tree displays only the business groups to which you belong.
+* Access or modify the parent business group's or root organization's settings
+* View the parent business group's client ID and client secret
 
-[[creatingroles]]
-== Permissions and Membership in Business Groups
+When you create a business group under another business group, only the redistributable entitlements, such as VPCs and load balancers, that were assigned to the parent business group can be allocated to the child business group.
 
-Permissions and roles are applied at the root organization level as well as at the business group level. However, each controls different resources. Permissions and roles assigned to a user in one particular business group are independent of the permissions and roles assigned to that same user in other business groups.
+Permissions and roles are applied at both the root organization and individual business group levels, but control access to different resources. Permissions and roles assigned to a user in one business group do not apply to that user's access in other business groups.
 
-To interact with a business group, you must be granted a permission within that business group, or be a member of a team that is granted permission within that business group. All users, regardless of which business groups they are in, belong to the root organization.
+To interact with a business group, a user must have a permission granted within that group or be a member of a team with permission in that group. All users belong to the root organization, regardless of business group membership.
 
 As a member of a business group, you are automatically granted permissions to view and access the business group, but to access certain resources, you must have the appropriate permissions or roles assigned to you. For example, to access APIs and CloudHub deployments in the business group, you must be assigned permissions that apply to that business group.
 
-For example, if you have the Organization Administrator permission in BusinessGroupA, you can grant users and teams access to APIs and CloudHub deployments in BusinessGroupA.
+For example:
 
-If you also have the Organization Administrator permission in BusinessGroupB, this does not mean that you can grant access to APIs and CloudHub deployments in BusinessGroupA to users in BusinessGroupB, and vice versa.
+*  If you have the Organization Administrator permission in BusinessGroupA, you can grant users and teams access to APIs and CloudHub deployments within BusinessGroupA.
+* That same permission in BusinessGroupB doesn't allow you to grant access to APIs and CloudHub deployments in BusinessGroupA.
 
 Resources that belong to the root organization require permissions specified at the root organization level.
```

Comment:
Says the same thing as line 38? Maybe fine to reiterate.

Reply:
Yeah, I think I'll leave it just in case.